app/views/session/delete_answers.scala.html

```diff
@@ -22,15 +22,20 @@
     implicit val viewConf: ViewConfig
 )
 
-@()(implicit request: Request[_], messages: Messages)
+@(signInAgain: String)(implicit request: Request[_], messages: Messages)
```

Comment:
it'd be nicer to pass in something which has a better type like `Call` maybe? Then it's harder to pass in the wrong thing

Reply:
Wrapped as `Call` as suggested